diff --git a/node_modules/react-native-paper/src/utils/forwardRef.tsx b/node_modules/react-native-paper/src/utils/forwardRef.tsx
index bca954b..0761712 100644
--- a/node_modules/react-native-paper/src/utils/forwardRef.tsx
+++ b/node_modules/react-native-paper/src/utils/forwardRef.tsx
@@ -19,5 +19,5 @@ export type ForwardRefComponent<T, P = {}> = ForwardRefExoticComponent<
  * More info: https://github.com/callstack/react-native-paper/pull/3603
  */
 export const forwardRef: <T, P = {}>(
-  render: ForwardRefRenderFunction<T, P>
+  render: ForwardRefRenderFunction<T, PropsWithoutRef<P>>
 ) => ForwardRefComponent<T, P> = React.forwardRef;
